<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>389</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>551</width>
    <height>389</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>551</width>
    <height>389</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icon.qrc">
    <normaloff>:/new/prefix1/icon.png</normaloff>:/new/prefix1/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(150, 225, 200);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lTitre">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>10</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; font-style:italic; text-decoration: underline;&quot;&gt;Calculator&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lENb2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lENb1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="lEqual">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;=&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBCalculatePlus">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
font: 12pt &quot;Comic Sans MS&quot;;
background-color: rgb(85, 200, 0);
border-radius: 10px;
}

QPushButton::hover {
background-color: rgb(200, 255, 100);
border-style: inset;
}

QPushButton::pressed {
background-color: rgb(255, 185, 50);
border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBClear">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(200, 200, 200);
border-radius: 10px;
}

QPushButton::hover {
background-color: rgb(220, 220, 220);
border-style: inset;
}

QPushButton::pressed {
background-color: rgb(150, 150, 150);
border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBExit">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>330</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
font: 12pt &quot;Comic Sans MS&quot;;
background-color: rgb(255, 85, 0);
border-radius: 10px;
}

QPushButton::hover {
background-color: rgb(255, 150, 25);
border-style: inset;
}

QPushButton::pressed {
background-color: rgb(255, 185, 50);
border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>370</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:7pt; font-style:italic;&quot;&gt;Â© PouletEnSlip - 2022&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lSum">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pBCalculateFois">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
font: 12pt &quot;Comic Sans MS&quot;;
background-color: rgb(85, 200, 0);
border-radius: 10px;
}

QPushButton::hover {
background-color: rgb(200, 255, 100);
border-style: inset;
}

QPushButton::pressed {
background-color: rgb(255, 185, 50);
border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBCalculateDiviser">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>190</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
font: 12pt &quot;Comic Sans MS&quot;;
background-color: rgb(85, 200, 0);
border-radius: 10px;
}

QPushButton::hover {
background-color: rgb(200, 255, 100);
border-style: inset;
}

QPushButton::pressed {
background-color: rgb(255, 185, 50);
border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBCalculateMoins">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>190</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
font: 12pt &quot;Comic Sans MS&quot;;
background-color: rgb(85, 200, 0);
border-radius: 10px;
}

QPushButton::hover {
background-color: rgb(200, 255, 100);
border-style: inset;
}

QPushButton::pressed {
background-color: rgb(255, 185, 50);
border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number 2</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Icon.qrc"/>
 </resources>
 <connections/>
</ui>
